main_routers/workshop_router.py

```
@@ -0,0 +1,1694 @@
+                return JSONResponse(content={"success": False, "error": f"缺少必要字段: {field}"}, status_code=400)
+        
+        # 提取数据
+        title = data['title']
+        content_folder = data['content_folder']
+        visibility = int(data['visibility'])
+        preview_image = data.get('preview_image', '')
+        description = data.get('description', '')
+        tags = data.get('tags', [])
+        change_note = data.get('change_note', '初始发布')
+        
+        # 规范化路径处理 - 改进版，确保在所有情况下都能正确处理路径
```

Ruff (0.14.8)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)